llvm/test/Transforms/LoopVectorize/AArch64/armpl-calls.ll

```diff
@@ -1,21 +1,23 @@
-; RUN: opt -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize -S < %s | FileCheck %s --check-prefixes=CHECK,NEON
-; RUN: opt -mattr=+sve -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize -S < %s | FileCheck %s --check-prefixes=CHECK,SVE
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "call.*(cos|sin|tan|cbrt|erf|^exp|gamma|log|sqrt|copysign|dim|min|mod|hypot|nextafter|pow|fma|mod)" --version 4
+; RUN: opt -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize,simplifycfg -prefer-predicate-over-epilogue=predicate-dont-vectorize -force-vector-interleave=1 -S < %s | FileCheck %s --check-prefix=NEON
+; RUN: opt -mattr=+sve -vector-library=ArmPL -passes=inject-tli-mappings,loop-vectorize,simplifycfg -prefer-predicate-over-epilogue=predicate-dont-vectorize -force-vector-interleave=1 -S < %s | FileCheck %s --check-prefix=SVE
```

Comment:
I got a warning after doing this
```
WARNING: Prefix CHECK had conflicting output from different RUN lines for all functions in test
```
Is there away to avoid this and generate common checks only for the LABEL of function definitions?

Reply:
Isn't the case that for the SVE case we are having attribute +sve added to the function, while for the neon case we won't? so the common CHECK lines for function definitions can not be used? also maybe the script changed so it is not possible to do it like that anymore.

Reply:
But the +sve attribute was added before, and those were still collapsing.
So it could be the 2nd case. I've discussed with @labrinea about the bump to `--version 4` (from `2) so this maybe have something to do with it.